modules/installation-and-upgrade/pages/container-deployment/mlm/migrations/proxy/proxy-mlm-50-51.adoc

```diff
@@ -98,38 +111,68 @@ Use the Web UI to migrate from:
 
 +
 
+[NOTE]
+====
+It is recommended to do a dry-run first before performing the actual migration.
+====
+
++
+
 . Monitor the Migration Action
 
 +
 
-. Reboot the System
+You can follow the process under menu:Systems[Details > Events] tab in the Web UI.
+
++
+
+. After the upgrade completes, stop the proxy container and then reboot the system
 
 +
 
 [source,console]
 ----
+mgrpxy stop
 reboot
 ----
 
 +
 
-. Post-Reboot Validation
+. Post-Reboot: Verify upgrade status
 
 +
 
 [source,console]
 ----
 cat /etc/os-release
+SUSEConnect --status-text
+----
+
++
+
+You should see:
+
+- `PRETTY_NAME="SUSE Linux Micro 6.1"`
+- `SUSE Multi-Linux Manager Proxy 5.1 Extension`
+
++
+
+. Verify {productname} Tools version
+
++
+
+[source,console]
+----
 mgrpxy --version
 ----
 
 +
 
-Expected:
+Expected output:
 
-- OS: `SL Micro 6.1`
-- Updated Proxy version
+- `mgrpxy version 5.1.11` or later
```

Comment:
```suggestion
- `mgrpxy version 5.1.0` or higher
```

Reply:
Fixed also on the server migration page. Thank you, Abid.